### ISSUE TITLE
Cannot read property 'get' of null in document view

### ISSUE THREAD

**Issue by SashaKolbasov** [avocode/avocode#1669]
* **Version**: 2.16.0
* **Sketch plugin version**: 47.0.0

* **Web/Desktop**: Desktop

* **Platform**: darwin

* **Avocode UID**: 198506
---

```
TypeError: Cannot read property 'get' of null
    at t._updateViewportSize (file:///Applications/%D0%A1%D0%BE%D1%84%D1%82/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3088460)
    at Immediate.<anonymous> (file:///Applications/%D0%A1%D0%BE%D1%84%D1%82/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3082713)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

**Comment by jakubzitny** [comment]
Hi @SashaKolbasov, thanks for reporting the problem. When is it happening exactly please? It is definitely a bug and we'll be fixing it for the next release. Any details on this would help though.

**Comment by SashaKolbasov** [comment]
I've never had this before. This happens, as I understand it, during the scrolling of the layout in the avocode or when changing the scale. Because of what it can be?

**Comment by SashaKolbasov** [comment]
Although not, the error appears at any action with the interface, after opening some strange sketch file.

**Comment by jakubzitny** [comment]
Hi, this has been fixed in latest Avocode `2.16.1`, please update and let us know if you have any more problems :wink:.
